tests/e2e-kubernetes/testsuites/credentials.go

```diff
@@ -465,7 +472,8 @@ func (t *s3CSICredentialsTestSuite) DefineTests(driver storageframework.TestDriv
 				})
 
 				It("should use up to date role associated with pod's service account", func(ctx context.Context) {
-					vol := createVolumeResourceWithMountOptions(enablePodLevelIdentity(ctx), l.config, pattern, []string{"allow-delete"})
+					var mountOptions = append([]string{"allow-delete"}, fmt.Sprintf("region %s", DefaultRegion))
```

Comment:
nit: You can write `mountOptions := []string{"allow-delete", fmt.Sprintf("region %s", DefaultRegion)}` instead of using `append` here